UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Logging/EventIds.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.Extensions.Logging;
+
+namespace UKHO.ExchangeSetService.Common.Logging
+{
+    public enum EventIds
+    {
+        SCSGetProductsFromSpecificDateRequestStart = 805000,
```

Comment:
As part of this PBI only LogRequest and ErrorRedactingResponseBody is being used. Other eventids are not referenced and part of other PBIs correct?(PBI#6850 and PBI#2750).

Reply:
Yes. Other eventids are part of other PBI. No need to remove.